src/Grid/topology.jl

```diff
@@ -14,6 +13,7 @@ the given entity is included in the returned list as well.
 """
 getneighborhood
 
+#TODO: Remove
```

Comment:
```suggestion
#TODO: To be removed
```
You mean like this, or to remove the comment? 
Currently, `EntityNeighborhood` is still used by the `vertex_star_stencils` code. 

I guess it would make sense to change the code for this as well to return an `ArrayOfVectorViews` instead of `Dict: global_vertex_nr -> Vector{EdgeIndex}`? I didn't so far to minimize the diff, and I don't know exactly the use-cases for that function.

Reply:
`vertex_star_stencils` is, if I remember correctly, giving you all vertices that are connected by an edge. It can be used for mixing FEM and FDM approaches as in e.g. the topology optimization, but there the FDM operator acts on the "elements", whereas you can change that to operate on the nodes (which I think is more accurate). Would be nice to remove that. Is it currently really returning an `EdgeIndex` vector?

Reply:
Yes, a view of a `Vector{EdgeIndex}`.